collections/programming-languages/index.md

```diff
@@ -49,7 +49,7 @@ items:
 - livecode/livecode
```

Comment:
[](url)

Reply:
@duc811997 sorry, I understand what you're commenting on here?